src/XMakeBuildEngine/UnitTests/Definition/ToolsetConfigurationReader_Tests.cs

```diff
@@ -581,12 +581,12 @@ public void ExtensionPathsTest_Basic1()
 #if XPLAT
```

Comment:
(discussion, doesn't need to be done now)

Should we remove this condition? It seems . . . imprecise, and I vaguely recall that it was mostly to reduce code churn caused by porting a feature that originated in xplat to master.

(I'm [working on it!](https://github.com/Microsoft/msbuild/milestones/The%20Great%20Merge))